_templates/sidebar_contact.html

```diff
@@ -13,6 +13,15 @@
             Developer guide</a></li>
 
   </ul>
+<h3>{{ _('GSoC') }}</h3>
+    <ul class="simple">
+        <li>
+            <a class="reference external" href="https://github.com/networkx/networkx/wiki/GSoC-2015">
+                GSoC 2015</a></li>
```

Comment:
Just a formatting thing: Why not indent `</li>` properly on a new line. Same for the latter.